join/join-testnet-with-binaries.sh

```diff
@@ -0,0 +1,165 @@
+AXELAR_MNEMONIC_PATH=""
+BIN_DIRECTORY="$ROOT_DIRECTORY/bin"
+AXELARD="$BIN_DIRECTORY/axelard"
+AXELARD_CMD="$AXELARD"
+OS="$(uname | awk '{print tolower($0)}')"
+ARCH="$(uname -m)"
+
+for arg in "$@"; do
+  case $arg in
+    --validator-mnemonic)
+    AXELAR_MNEMONIC_PATH="$2"
+    shift
+    ;;
+    --tendermint-key)
+    TENDERMINT_KEY_PATH="$2"
+    shift
+    ;;
+    --reset-chain)
+    RESET_CHAIN=true
+    shift
+    ;;
+    -r|--root)
+    ROOT_DIRECTORY="$2"
+    shift
+    ;;
+    --axelar-core)
+    AXELAR_CORE_VERSION="$2"
+    shift
+    ;;
+    --bin-directory)
+    BIN_DIRECTORY="$2"
+    shift
+    ;;
+    *)
+    shift
+    ;;
+  esac
+done
+
+
+if [ "$(ps aux | grep -c '[a]xelard start --home')" -gt "0" ]; then
+  echo "Node already running. Run 'killall axelard' to kill node.";
+  exit 1;
+fi
+
+if [ -z "$AXELAR_CORE_VERSION" ]; then
+  echo "'--axelar-core vX.Y.Z' is required"
+  exit 1
+fi
+
+echo "Axelar Core Version: ${AXELAR_CORE_VERSION}"
+echo "OS: ${OS}"
+echo "Architecture: ${ARCH}"
+echo "Root Directory: ${ROOT_DIRECTORY}"
+
+if $RESET_CHAIN; then
+  rm -rf "$ROOT_DIRECTORY"
+  rm -rf "$TOFND_DIRECTORY"
+  rm -rf ~/.axelar
+fi
+
+mkdir -p "$ROOT_DIRECTORY"
+mkdir -p "$BIN_DIRECTORY"
+
+LOGS_DIRECTORY="${ROOT_DIRECTORY}/logs"
+mkdir -p "$LOGS_DIRECTORY"
+
+CONFIG_DIRECTORY="${ROOT_DIRECTORY}/config"
+mkdir -p "$CONFIG_DIRECTORY"
+
+CORE_DIRECTORY="${ROOT_DIRECTORY}/.core"
+mkdir -p "$CORE_DIRECTORY"
+
+AXELARD_BINARY="axelard-${OS}-${ARCH}-${AXELAR_CORE_VERSION}"
+if [ ! -f "${AXELARD}" ]; then
+  echo "Downloading axelard binary $AXELARD_BINARY"
+  curl -s https://axelar-releases.s3.us-east-2.amazonaws.com/axelard/${AXELAR_CORE_VERSION}/${AXELARD_BINARY} -o "${AXELARD}" && chmod +x "${AXELARD}"
+fi
+
+if [ ! -f "${CONFIG_DIRECTORY}/genesis.json" ]; then
+  echo "Downloading genesis.json"
+  curl -s https://axelar-testnet.s3.us-east-2.amazonaws.com/genesis.json -o "${CONFIG_DIRECTORY}/genesis.json"
+fi
+
+if [ ! -f "${CONFIG_DIRECTORY}/peers.txt" ]; then
+  echo "Downloading peers.txt"
+  curl -s https://axelar-testnet.s3.us-east-2.amazonaws.com/peers.txt -o "${CONFIG_DIRECTORY}/peers.txt"
+fi
+
+if [ ! -f "${CONFIG_DIRECTORY}/config.toml" ]; then
+  echo "Moving config.toml to config directory"
+  cp "${GIT_ROOT}/join/config.toml" "${CONFIG_DIRECTORY}/config.toml"
+fi
+
+if [ ! -f "${CONFIG_DIRECTORY}/app.toml" ]; then
+  echo "Moving app.toml to config directory"
+  cp "${GIT_ROOT}/join/app.toml" "${CONFIG_DIRECTORY}/app.toml"
+fi
+
+addPeers() {
+  echo "Adding peers to config.toml"
+  sed "s/^seeds =.*/seeds = \"$1\"/g" "$CONFIG_DIRECTORY/config.toml" >"$CONFIG_DIRECTORY/config.toml.tmp" &&
+  mv "$CONFIG_DIRECTORY/config.toml.tmp" "$CONFIG_DIRECTORY/config.toml"
+}
+
+addPeers "$(cat "${CONFIG_DIRECTORY}/peers.txt")"
+
+export NODE_MONIKER=${NODE_MONIKER:-"$(hostname)"}
+export AXELARD_CHAIN_ID=${AXELARD_CHAIN_ID:-"axelar"}
+
+echo "Node moniker: $NODE_MONIKER"
+echo "Axelar Chain ID: $AXELARD_CHAIN_ID"
+set -x
```

Comment:
will we need to leave this here?

Reply:
no will remove once everything is working 100%